frontend/react-app/test/dijkstra/dijkstra.test.js

```diff
@@ -0,0 +1,5 @@
+test(
```

Comment:
ファイルの場所は`react-scripts test`を使う場合に限った話  
`jest`でいくならどこでも行けると思う

Reply:
f85012388c7bf8796e86c052a14a0bfb9256e423
にて `src/dijkstra` にテストを移動

※公式ドキュメントに「テストはソースコードを同じディレクトリに作るのがよい」と書いてあるのでそれにならった
https://create-react-app.dev/docs/running-tests/
> We recommend to put the test files (or __tests__ folders) next to the code they are testing so that relative imports appear shorter.